blog/guide.ftd

```diff
@@ -0,0 +1,300 @@
+well if you've got a team of writers contributing to your blog.
+
+-- ds.h3: [**`blog-articles`**](https://github.com/fastn-community/saurabh-portfolio/tree/main/blog-articles) folder
+
+Each fresh blog post gets its own `.ftd` file saved right in this folder. 
+Keeps everything tidy and organized for your blog content.
+
+-- ds.h1: Getting Started with Customizing the Template
+
+Here are a few prerequisites to get started with a fastn template.
+
+-- ds.h2: Step 1: Setting Up Tools and Accounts
+
+[Set up your GitHub account](https://github.com/join) if you haven’t already.
+
+-- note.note: Since fastn is open-source, you can find the source code of all components and templates on the [fastn community on GitHub](https://github.com/fastn-community).
+
+-- ds.markdown:
+
+To create your copy of the template, click `fork repository`. If you are like 
+me and already have fastn, GitHub desktop, and a text editor installed, simply 
+copy the HTTPS URL of the template. Then, go to your GitHub desktop, clone the 
+repository, and choose where to save it on your local system.
+
+An alternative way to use the template is by utilizing GitHub Pages to host 
+your built assets. Follow the steps detailed in this guide 
+[Publishing Static Site On GitHub Pages](https://fastn.com/github-pages/).
+
+-- ds.h2: Step 2: Adding Assets
+
+Open the assets folder and start adding images, icons, and other graphical 
+elements intended for your website. 
+
+-- note.note: When saving graphics, remember to use small caps and hyphens between words for better organization and access. 
+
+Example: `profile-picture.png`
+
+-- ds.h2: Step 3: Editing the Template
+
+I opened the template folder on the text editor. Began by editing the `index.ftd` 
+file and moved to the `common.ftd` file.
+
+-- ds.h3: Editing the `index.ftd` file
+
+1) I removed sections or components I won’t be using on my website. 
+
+Example: 
+
+- `Docs` section
+- `-- info-list:` from the `-- about-me-info:` section.
+- `My Skills` section
+- Addresses in the contact form
+- Social Links like Facebook, Twitter and Instagram.
+- And other content elements wherever it is not needed.
+
+2) I modified the content elements like titles, tags, sub-titles, body text, 
+CTA button texts, links, and images. Here is my modified hero section:
+
+-- ds.code:
+lang: ftd 
+
+\-- hero: Are you in the business of saving the planet?
+tag: Copywriting for Sustainable Businesses
+avatar: $assets.files.assets.profile-picture.jpeg
+cta-primary: My Work
+cta-primary-link: /#latest-works
+cta-secondary: Book a Meeting
+cta-secondary-link: https://calendly.com/nandhini-copywriter/discovery-call/
+
+-- ds.markdown:
+
+If you're someone who doesn't just sell stuff but rallies behind 
+a cause, I'll whip up words that'll click with your audience and 
+turn them into full-on cheerleaders for the planet and your business.
+
+-- ds.h3: Editing the index.ftd file in the common folder
```

Comment:
I think we can ask Meenu to split the index file into multiple files, like `testimonials.ftd` etc for quicker access, you will not have to scroll much to find what you are looking for etc. @Arpita-Jaiswal please help here.

Reply:
@MeenuKumari28